### ISSUE TITLE
Port kernel dist to local static web server instead of remote kernel

### ISSUE THREAD

**Issue by kuruk-mm** [decentraland/unity-renderer#674]
The Desktop Client must be distributed with the kernel dist on it.

So we need to put the files with the build and create a local static webserver to host those files.

**Comment by kuruk-mm** [comment]
Blocked by decentraland/explorer#2473

**Comment by olavra** [comment]
This task is no longer needed as we will run kernel in nodejs for the desktop distribution
